storage/store.go

```diff
@@ -1199,6 +1199,15 @@ func (s *Store) Descriptor() (*proto.StoreDescriptor, error) {
 // method, args & reply into a Raft Cmd struct and executes the
 // command using the fetched range.
 func (s *Store) ExecuteCmd(ctx context.Context, call proto.Call) error {
+	return s.ExecuteCmdWithCallback(ctx, call, nil)
+}
+
+// ExecuteCmdWithCallback is called from ExecuteCmd. When a debug
+// callback is not nil, it will be called whenever the command is
+// executed (and retried).
+//
+// The method is exposed for testing.
+func (s *Store) ExecuteCmdWithCallback(ctx context.Context, call proto.Call, dbgCallback func()) error {
```

Comment:
Ah, didn't know about `TestingCommandFilter`.. Fixed the code and reverted the change to `store.go`.